### ISSUE TITLE
ProgressBar broken

### ISSUE THREAD

**Issue by mirand863** [p-ranav/indicators#89]
What could possibly be causing the progress bar to behave this way?

![image](https://user-images.githubusercontent.com/22525107/110041196-295ad980-7d44-11eb-8968-4f9aefa2f6ec.png)

PS: I only updated the output to cerr instead of cout, but it should still work, right? If I use cout it doesn't print anything at all.

**Comment by p-ranav** [comment]
* Can you paste the relevant code here? 
* Can you tell me what OS + terminal you are using?

**Comment by p-ranav** [comment]
Hello, can you check if this commit has fixed the problem? 4790bfd

**Comment by mirand863** [comment]
@p-ranav Thank you for the response. I have fixed it by adding a carriage return in this bit of the code:

```
    const auto postfix_pair = get_postfix_text();
    const auto postfix_text = postfix_pair.first;
    const auto postfix_length = postfix_pair.second;
    os << postfix_text << "\r";
```

I will try the new version of your code as well, thank you!
